src/App.php

```diff
@@ -5,8 +5,6 @@
 use Zend\Diactoros\Response;
 use GianArb\Penny\Event\HttpFlowEvent;
 use DI\ContainerBuilder;
-use Psr\Http\Message\ResponseInterface;
-use Psr\Http\Message\RequestInterface;
```

Comment:
@sensorario at the moment I don't use request and response into the framework, why should it constraint itself? :)